doc/manual/parallel-computing.rst

```diff
@@ -498,6 +561,66 @@ the backing store on the remote process.
 example of this is provided in ``examples/dictchannel.jl`` which uses a dictionary as its remote store.
 
 
+Channels and RemoteChannels
+---------------------------
+- A :class:`Channel` is local to a process.
+- A :class:`RemoteChannel` can be thought of as a *handle* to a :class:`Channel`.
+- The process id, ``pid``, associated with a :class:`RemoteChannel` identifies the process where
+  the backing store, i.e., the backing :class:`Channel` exists.
+- Any process with a reference to a :class:`RemoteChannel` can put and take items from the channel.
+  Data is automtically sent to (or retrieved from) the process a :class:`RemoteChannel` is associated with.
+- Serializing and deserializing a :class:`Channel` also serializes/deserializes any data
```

Comment:
Perhaps include a link here to the part of the manual that talks about serialization?